tests/integration_tests/security/test_spectre_meltdown.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# N.B.: Although this repository is released under the Apache-2.0, part of its test requires a
+# script from the third party "Spectre & Meltdown Checker" project. This script is under the
+# GPL-3.0-only license.
+"""Tests spectre / meltdown mitigation."""
+
+import pytest
+import requests
+
+from framework import utils
+from host_tools.network import SSHConnection
+
+
+CHECKER_URL = "https://meltdown.ovh"
+CHECKER_FILENAME = "spectre-meltdown-checker.sh"
+
+
+@pytest.fixture(name="spectre_meltdown_checker")
+def download_spectre_meltdown_checker(tmp_path):
+    """Download spectre / meltdown checker script."""
+    resp = requests.get(CHECKER_URL, timeout=5)
+    assert (
+        resp.ok
+    ), f"Failed to download the checker script. Status code: {resp.status_code}"
+
+    path = tmp_path / CHECKER_FILENAME
+    with open(path, "wb") as file:
+        file.write(resp.content)
+
+    return path
+
+
+def test_spectre_meltdown_checker_on_host(spectre_meltdown_checker):
+    """
+    Test with the spectre / meltdown checker on host.
+
+    @type: security
+    """
+    utils.run_cmd(f"sh {spectre_meltdown_checker} --explain")
+
+
+def test_spectre_meltdown_checker_on_guest(
+    spectre_meltdown_checker,
+    test_microvm_with_spectre_meltdown,
+    network_config,
+):
+    """
+    Test with the spectre / meltdown checker on guest.
+
+    @type: security
+    """
+    microvm = test_microvm_with_spectre_meltdown
+    microvm.spawn()
+    microvm.basic_config()
+    microvm.ssh_network_config(network_config, "1")
+    microvm.start()
+
+    run_spectre_meltdown_checker_on_guest(
+        microvm,
+        spectre_meltdown_checker,
+    )
+
+
+def test_spectre_meltdown_checker_on_guest_with_template(
+    spectre_meltdown_checker,
+    test_microvm_with_spectre_meltdown,
+    network_config,
+    cpu_template,
```

Comment:
Is this a fixture? where is it defined?

Reply:
`cpu_template` is defined in `conftest.py` introduced in this commit d142849309373c2cd654e727ba1d22879293fb0b.